packages/reflect-yjs/src/provider.ts

```diff
@@ -23,33 +22,25 @@ export class Provider {
     ydoc.on('update', this.#handleUpdate);
     ydoc.on('destroy', this.#handleDestroy);
 
-    this.#cancelUpdateSubscribe = reflect.subscribe(
+    this.#cancelSubscribe = reflect.subscribe(
```

Comment:
This merging of the subscribes is an important fix independent of the chunking change.  Testing the chunking change revealed the problem to me.

When loading a large document the first subscription here would run calling `Y.applyUpdateV2(ydoc, update);,` which would fire an 'update' event on the ydoc, causing us to call 

```
    const diffUpdate = this.#vector
      ? Y.encodeStateAsUpdateV2(this.#ydoc, this.#vector)
      : Y.encodeStateAsUpdateV2(this.#ydoc);
    await this.#reflect.mutate.updateYJS({
      name: this.name,
      update: base64.fromByteArray(diffUpdate),
    });
   ```
   
However the above is happening before the second subscribe body has a chance to run, and so this.#vector is not updated, so we  compute the update off of an out of date this.#vector (possibly even undefined) result in a much larger than necessary update.

Reply:
I was actually thinking about this last night. "Why are there two subscriptions?"

Thanks for fixing